src/osa/scripts/gainsel_webmaker.py

```diff
@@ -130,9 +130,10 @@ def main():
         directory = Path(cfg.get("LST1", "GAIN_SELECTION_FLAG_DIR"))
 
         directory.mkdir(parents=True, exist_ok=True)
-
-        html_file = directory / f"osa_gainsel_status_{flat_date}.html"
-        html_file.write_text(html_content(html_table, flat_date, "OSA Gain Selection"), encoding="utf-8")
```

Comment:
is the write_text (I think from Path?) not working as expected?

Reply:
from docs 

    Open the file pointed to in text mode, write data to it, and close the file:

Reply:
I found some issues when it came to create a new file, so I used that code as an alternative